documentation/src/main/asciidoc/userguide/appendices/Annotations.adoc

```diff
@@ -7,45 +7,45 @@
 [[annotations-jpa-access]]
 ==== `@Access`
 
-The http://docs.oracle.com/javaee/7/api/javax/persistence/Access.html[`@Access`] annotation is used to specify the access type of the associated entity class, mapped superclass, or embeddable class, or entity attribute.
+The https://javaee.github.io/javaee-spec/javadocs//javax/persistence/Access.html[`@Access`] annotation is used to specify the access type of the associated entity class, mapped superclass, or embeddable class, or entity attribute.
```

Comment:
You introduced double `/` in the URIs.

Please replace `https://javaee.github.io/javaee-spec/javadocs//` with `https://javaee.github.io/javaee-spec/javadocs/` in the docs.

Reply:
Interesting that the links work just fine. I'll rerun the replace pattern tomorrow and push it.

Reply:
Yeah, either web servers or browsers fix that for you.